template/pkg/init.sls

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+include:
+  - .install
```

Comment:
Do we need `install.sls` because as general rule I prefer using `init.sls` for installation stuff to mirror fact that `clean.sls` removes stuff?  I would prefer `install.sls` be removed for the simple scenario.

Reply:
For consistency, let's leave all `init.sls` doing nothing more than directing traffic.  So it becomes easier to see the flow of the formula by opening up the `init.sls` files and seeing how they're linked.

Reply:
That's why I've also limited these files to use relative includes, so that they don't get abused for anything else.  Not even defining `topdir`.

Reply:
okay, but following this logic `clean.sls` would needs to direct traffic too ;-)

Reply:
@aboe76 @javierbertoli  For subdirectories I prefer `directory/init.sls`  &  `directory/clean.sls` to include SLS states and do work so that init.sls and clean.sls mirror each other.

@myii prefers `init.sls` to exclude SLS and only have includes.  Do you guys have a preference?

Reply:
I agree with `init.sls` only have includes

Reply:
@daks  For consistency should `clean.sls` only have includes too?